app/components/DisplayContent/index.js

```diff
@@ -1,7 +1,11 @@
 //@flow
 
 import React from 'react';
-import Editor from 'app/components/Editor';
+import Editor from '@webkom/lego-editor';
+import '@webkom/lego-editor/dist/Editor.css';
```

Comment:
Is there a reason to include these twice, and not just directly in `index.js`?

Reply:
I just think it makes more sense to have them beside the component.